### ISSUE TITLE
Tradepile Error - FUTBUD

### ISSUE THREAD

**Issue by M-Denz** [Lululuz/FutBud#6]
Platform - Windows
Platform Version - Windows 7 Professional
Architecture - x64

FUTBUD Platform - PS4/PS3
Tried - Emptying Transfer list off any current items, only receiving items brought from FUTBUD. Adjusted search and checking trade-pile times, buy less than x player etc. Debugging On.

Hello, whenever FUTBUD successfully buys any player to a list, as soon as it checks tradepile, it returns an tradeplie error. Also when it checks if the list already contains players it returns the error.

I'll upload a picture in a bit.

Thank you :)

M_Denz

8/12 EDIT (PS3): Thats weird, I think when I saved the list somewhere and loaded it again solved the issue. Also, I sure I already turned off and on the Debug button when the issue was present
8/12 EDIT EDIT (PS3): Spoke too soon, after a while it starts stating the Tradepile Error again. I'll look it it into more and get back to you.

**Comment by Lululuz** [comment]
Please let me know if you find something

**Comment by M-Denz** [comment]
Ok, new update!
The error just never seems to go away on PS4, and that only makes it able to buy players not sell. The PS3 however comes, then goes, then comes again and the second time it comes it stays there until restart. I've also tried clearing the transfer list of all the players except the ones brought but makes no difference. I even cleared the list completely before I started the app but as soon as it buys a player for any of these platforms, it returns the tradeplie error. I also tried changing the "Counter" section of the players to match the current players in my list but no luck.
